OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -140,17 +191,35 @@ $ sudo chown 1000:1000 /scratch/wcpdocker/volumes/es
 #### B. Creating and Running Elasticsearch Container
 
 ```
-$ docker run -i -t --name ESContainer --network=WCPortalNET -p 9200:9200 --volumes-from WCPortalContainer -v $ES_DATA_VOLUME:/u01/esHome/esNode/data --env-file <directory>/webcenter.env.list $WCPortalImageName configureOrStartElasticsearch.sh
+$ docker run -i -t --name <ES_CONTAINER_NAME> --network=<NETWORK_NAME> -p <HostFreePort>:9200 --volumes-from <WCP_CONTAINER_NAME> -v <ES_DATA_VOLUME>:/u01/esHome/esNode/data --env-file webcenter.env.list oracle/wcportal:12.2.1.4.0 configureOrStartElasticsearch.sh
 ```
+Sample Run Command
 
+```
+$ docker run -i -t --name ESContainer --network=WCPortalNET -p 9200:9200 --volumes-from WCPortalContainer -v /scratch/wcpdocker/volumes/es:/u01/esHome/esNode/data --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4.0 configureOrStartElasticsearch.sh
+
+```
+
+Elasticsearch Container start up command explained:
+
+| 		Parameter    	 |     Parameter Name      | 							     		Description			                               |
+| :--------------------: | :---------------------: | :---------------------------------------------------------------------------------------: |
+| --name                 | ES_CONTAINER_NAME          |  Set to Elasticsearch  Container Name                                                |
+| --network              | NETWORK_NAME            | User-defined network to connect to; use the one created earlier ‘WCPortalNET’.             |
+| -p                     | HostFreePort:9200| ElasticSearch default port 9200 as container port , Maps the container port to host's  port.              	   |
+| --volumes-from         | WCP_CONTAINER_NAME  | Set the WCP Portal Container Name               	   |
+| -v                    | ES_DATA_VOLUME| Mounts the host directory as a Volume.      |
+| --env-file             | webcenter.env.list | `webcenter.env.list`  sets the environment variables.                            |
+| oracle/wcportal:12.2.1.4.0      | REPOSITORY:TAG      | Repository name:Tag name of the image created using buildDockerImage.sh  .   |
```

Comment:
Remove "created using buildDockerImage.sh" and fix up the wording as noted before.